.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 # Code Owners
 *   @willjwon
+*   @zli669
```

Comment:
Can you put your handle in line 2? Just like as in: https://github.com/astra-sim/astra-sim/blob/master/CODEOWNERS

Reply:
Done.